content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: blog
+title: 'Kubernetes v1.30: TBA'
+date: 2024-04-17
+slug: kubernetes-v1-30-release
+---
+
+**Authors:** [Kubernetes v1.30 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release_team.md)
+
+**Editors:** Amit Dsouza, Frederick Kautz, Kristin Martin, Abigail McCarthy, Natali Vlatko
```

Comment:
(aside) This could be hard to localize. How do we explain the difference between the authors of the article and the editors of the article, so that localization teams can pick the right word?

Reply:
We did the same with the [1.29 Release Blog](https://kubernetes.io/blog/2023/12/13/kubernetes-v1-29-release/). I'll check if there were any localization issues with that one.

Reply:
No issues noted by localization team member with the previous blog that used the same byline format